Gemfile.lock

```diff
@@ -73,16 +74,32 @@ GEM
     nokogiri (1.10.9)
       mini_portile2 (~> 2.4.0)
     parallel (1.19.1)
+    parser (2.7.1.3)
+      ast (~> 2.4.0)
     pathutil (0.16.2)
       forwardable-extended (~> 2.6)
     public_suffix (4.0.5)
+    rainbow (3.0.0)
```

Comment:
This doesn't seem to be a dependency for `rubocop`, could you please elaborate?

Reply:
Line 38 of the gemspec for Rubocop shows this.